parm/config/config.fcst

```diff
@@ -93,45 +93,45 @@ export new_o3forc="YES"
 # export launch_level=$(echo "$LEVS/2.35" |bc)
 
 # Sponge layer settings
-export tau=10.0
-export rf_cutoff=7.5e2
-export d2_bg_k1=0.20
-export d2_bg_k2=0.04
-export dz_min=2
-export n_sponge=42
 if [ $LEVS = "128" -a "$CDUMP" = "gdas" ]; then
-   export tau=5.0
-   export rf_cutoff=1.0e3
-   export d2_bg_k1=0.20
-   export d2_bg_k2=0.0
+   export tau=${tau:-5.0}
+   export rf_cutoff=${rf_cutoff:-1.0e3}
+   export d2_bg_k1=${d2_bg_k1:-0.20}
+   export d2_bg_k2=${d2_bg_k2:-0.0}
 fi
+export tau=${tau:-10.0}
+export rf_cutoff=${rf_cutoff:-7.5e2}
+export d2_bg_k1=${d2_bg_k1:-0.20}
+export d2_bg_k2=${d2_bg_k2:-0.04}
+export dz_min=${dz_min:-2}
+export n_sponge=${n_sponge:-42}
 
 # PBL/turbulance schemes
-export hybedmf=".false."
-export satmedmf=".true."
-export isatmedmf=1
-tbf=""
+export hybedmf=${hybedmf:-".false."}
```

Comment:
`variable=${variable:-default_value}`
implies one can define this variable before this script.
This being a configuration script then looses the point of being a configuration script, rather become a configurable script.

The syntax at this level should remain at key-value pairs.